stackdio/core/serializers.py

```diff
@@ -29,6 +30,14 @@ class StackdioHyperlinkedModelSerializer(serializers.HyperlinkedModelSerializer)
     """
     Override to use the appropriately namespaced url
     """
+
+    def add_extra_kwargs(self, kwargs):
```

Comment:
Since this is `kwargs`, do we want to use the standard `**kwargs` pattern here, or to force passing a dictionary?